package.json

```diff
@@ -31,12 +31,14 @@
     "web3-typescript-typings": "^0.7.2"
   },
   "dependencies": {
-    "chalk": "^2.1.0",
+    "bignumber.js": "^5.0.0",
```

Comment:
bignumber should be peer dependency

Reply:
I think it should be both as the project needs it and downstream projects need. Is that how peerDependencies work? You've got web3 as both a dependency and a peerDependency so I guess so.